htmresearch/algorithms/apical_tiebreak_bayesian_temporal_memory.py

```diff
@@ -133,19 +133,19 @@ def __init__(self,
     self.basalBias = np.zeros((self.numBasalSegments, self.columnCount*self.cellsPerColumn))
     self.apicalBias = np.zeros((self.numApicalSegments, self.columnCount*self.cellsPerColumn))
 
-    # Initialize moving averages with 1/(M_i*M_j), 1/(M_i) or 1/(M_j)
+    # Initialize moving averages uniformly with 1/(M_i*M_j), 1/(M_i) or 1/(M_j)
     self.basalMovingAverages = np.full(
-      (self.numBasalSegments, self.columnCount*self.cellsPerColumn, self.basalInputSize),
-      1.0/(self.numberOfCells()*self.basalInputSize)
+      (self.numBasalSegments, self.numberOfCells(), self.basalInputSize),
+      0.0
     )
     self.apicalMovingAverages = np.full(
-      (self.numApicalSegments, self.columnCount*self.cellsPerColumn, self.apicalInputSize),
-      1.0/(self.numberOfCells()*self.apicalInputSize)
+      (self.numApicalSegments, self.numberOfCells(), self.apicalInputSize),
+      0.0
     )
-    self.basalMovingAveragesBias = np.full((self.numBasalSegments, self.numberOfCells()), 1.0)
-    self.apicalMovingAveragesBias = np.full((self.numApicalSegments, self.numberOfCells()), 1.0)
-    self.basalMovingAverageInput = np.full(self.basalInputSize, 1.0)
-    self.apicalMovingAverageInput = np.full(self.apicalInputSize, 1.0)
+    self.basalMovingAveragesBias = np.full((self.numBasalSegments, self.numberOfCells()), 0.0)
```

Comment:
Same. 
That means tho that the initialisation had little or no effect with the new normalisation?

Reply:
It did have some effect but it lowered the probability in the activations (which we then pass to l2) if I remember correctly, thats why I set it down again. Could be tested around with when the CP works